Source/src/core/Scene.cpp

```diff
@@ -26,7 +26,7 @@ Hachiko::Scene::Scene()
     , name(UNNAMED_SCENE)
 {
     // TODO: Send hardcoded values to preferences
-    quadtree->SetBox(AABB(float3(-100, -100, -100), float3(100, 250, 100)));
+    quadtree->SetBox(AABB(float3(-100000000, -100000000, -100000000), float3(100000000, 100000000, 100000000)));
```

Comment:
Having a quadtree that its too bight might be dangerous, i would put reasonable values